src/frontend/tsx/Chessboard.tsx

```diff
@@ -56,11 +56,11 @@ export function Chessboard () {
       ? Side.White
       : Side.Black
     : getCgColor(chess.turn()) as Side
-  const isViewOnly = (displayIndex == null)
+  const isViewOnly = (displayIndex !== null && displayGame.archived) || (displayIndex == null)
     ? false
-    : (displayGame.info.moves == null)
+    : (displayGame.moves == null)
       ? false
-      : (displayIndex < displayGame.info.moves.length - 1)
+      : (displayIndex < displayGame.moves.length - 1)
```

Comment:
This conditional has become all kinds of borked, and I think I contributed to that last review - let's reset:
```
const isViewOnly = displayGame.archived
  || ((displayIndex !== null)
    && (displayGame.moves !== null)
    && (displayGame.moves.length > 0)
    && ((displayGame.moves.length - 1) > displayIndex))
```